edx_api/certificates/__init__.py

```diff
@@ -0,0 +1,75 @@
+"""
+edX Certificates REST API client class
+"""
+from requests.exceptions import HTTPError
+from six.moves.urllib.parse import urljoin  # pylint: disable=import-error
+
+from edx_api.enrollments import CourseEnrollments
+from .models import Certificate, Certificates
+
+
+class UserCertificates(object):
+    """
+    edX student certificates client
+    """
+
+    def __init__(self, requester, base_url):
+        """
+        Args:
+            requester (Requester): an authenticated objects for requests to edX
+            base_url (str): string representing the base URL of an edX LMS instance
+        """
+        self.requester = requester
+        self.base_url = base_url
+
+    def get_student_certificate(self, username, course_id):
+        """
+        Returns an Certificate object with the user certificates
+
+        Args:
+            username (str): an edx user's username
+            course_id (str): an edX course id.
+
+        Returns:
+            Certificate: object representing the student certificate for a course
+        """
+        # the request is done in behalf of the current logged in user
+        resp = self.requester.get(
+            urljoin(
+                self.base_url,
+                '/api/certificates/v0/certificates/{username}/courses/{course_key}/'.format(
+                    username=username,
+                    course_key=course_id
```

Comment:
Do we need to do any escaping of the course key or username? Would either ever have a slash?